level_5/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+set(
+	SRC_FILES
+	./main.cpp
+	./lexer.hpp
+	./interpreter.hpp
+)
+set(
+	EXEC_NAME
+	level_5
+)
+add_executable(
+	${EXEC_NAME}
+	${SRC_FILES}
+)
+add_compile_options(
+	${EXEC_NAME}
+	PRIVATE "-Wall" "-g"
```

Comment:
these flags should be either global or not defined at all (-Wall).

With respect to "-g", this is what you have `CMAKE_BUILD_TYPE` for, so it should not be needed anywhere in here.

Reply:
Done